.github/workflows/buildtracker.yml

```diff
@@ -0,0 +1,33 @@
+# Workflow that only runs when a commit lands on master.
+
+name: 👷‍♀️
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  buildtracker:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: git clone
+      uses: actions/checkout@v2
+
+    - name: Use Node.js 10.x
+      uses: actions/setup-node@v1
+      with:
+        node-version: 10.x
+
+    - run: yarn --frozen-lockfile
+    - run: yarn build-all
+
+    - name: Store in buildtracker
+      run: yarn bt-cli upload-build
+      env:
+        # https://buildtracker.dev/docs/guides/github-actions#configuration
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
I think `GITHUB_TOKEN` is already a set env variable.

Reply:
good call.